simulation/traffic_simulator/include/traffic_simulator/math/hermite_curve.hpp

```diff
@@ -72,8 +71,11 @@ class HermiteCurve
 private:
   std::pair<double, double> get2DMinMaxCurvatureValue() const;
   double length_;
+  boost::optional<double> solveNewtonMethod(
+    const geometry_msgs::msg::Point & point, double initial_value, double tolerance,
+    unsigned int max_iteration) const;
```

Comment:
`unsigned int` => `std::size_t`

https://stackoverflow.com/questions/131803/unsigned-int-vs-size-t

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/603/commits/71e2f34e3d6f0fa135e9708cba12d90aa0421824